rollup.config.js

```diff
@@ -0,0 +1,55 @@
+import nodeResolve from 'rollup-plugin-node-resolve';
+import commonjs from 'rollup-plugin-commonjs';
+import babel from 'rollup-plugin-babel';
+import uglify from 'rollup-plugin-uglify';
+import { sizeSnapshot } from 'rollup-plugin-size-snapshot';
+
+const input = './src/index.js';
+
+const umdGlobals = {
```

Comment:
Popper.js should be marked as external as well.

Reply:
Does it matter? Do you think somebody will use popper.js and react-popper in one script? For react-popper consumer there will be another action to add popper dependency.

Reply:
I don't want to get every week a new issue with someone that asks me to release a new UMD bundle with the new Popper.js version.